src/pip/_internal/vcs/subversion.py

```diff
@@ -33,6 +37,36 @@ def should_add_vcs_url_prefix(cls, remote_url):
     def get_base_rev_args(rev):
         return ['-r', rev]
 
+    def get_vcs_version(self):
+        # type: () -> Optional[Tuple[int, ...]]
+        """Return the version of the currently installed Subversion client.
+
+        :return: A tuple containing the parts of the version information or
+            ``None`` if the version returned from ``svn`` could not be parsed.
+        :raises: BadCommand: If ``svn`` is not installed.
+        """
+        version_prefix = 'svn, version '
+        version = self.run_command(['--version'], show_stdout=False)
+        if version.startswith(version_prefix):
```

Comment:
This would be a bit cleaner if you handled the "not" case first and returned early. Then you wouldn't need the `else` clause.